src/hypothesis/internal/conjecture/engine.py

```diff
@@ -292,7 +295,9 @@ def draw_constant(data, n, distribution):
             draw_new,
             reuse_existing, reuse_existing,
             draw_existing, draw_smaller, draw_larger,
-            flip_bit, draw_zero, draw_constant,
+            flip_bit,
+            draw_zero, draw_max, draw_zero, draw_max,
```

Comment:
Intentional - it's a deliberate bias to make it more likely (same as `reuse_existing` above).